dockerfiles/alpine/obam/Dockerfile

```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk8:jdk8u242-b08-alpine
+FROM adoptopenjdk/openjdk8:jdk8u272-b10-alpine
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
-      com.wso2.docker.source="https://github.com/wso2/docker-open-banking/releases/tag/v1.5.0.2"
+      com.wso2.docker.source="https://github.com/wso2/docker-open-banking/releases/tag/v2.0.0"
```

Comment:
@nalakagunathilaka this version will be `v2.0.0.1`. Please apply the change to all files.

Reply:
Resolved with latest commit.